protobuf/api/execution.proto

```diff
@@ -70,6 +70,12 @@ message CreateExecutionRequest {
     (gogoproto.customtype) = "github.com/mesg-foundation/engine/hash.Hash",
     (gogoproto.nullable) = false
   ];
+
+  // price of running the execution.
+  string price = 10 [
```

Comment:
Are you sure about this type in string?
I would import and use this type https://github.com/cosmos/cosmos-sdk/blob/master/types/types.proto#L14